Makefile

```diff
@@ -1,6 +1,11 @@
 NAME    := argobots
 SRC_EXT := gz
 SOURCE  = https://github.com/pmodels/$(NAME)/releases/download/v$(VERSION)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
-PATCHES  = $(NAME)-9d48af0840.patch $(NAME)-9d48af0840...89507c1f8c.patch
-
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
+PATCHES  = $(NAME)-9d48af0840.patch
+ifneq ($(ID_LIKE),debian)
+PATCHES += $(NAME)-9d48af0840...89507c1f8c.patch
```

Comment:
Is there any reason this patch can't be used for the non-debian builds also and we can do away with all of this distro based gating?

Reply:
None that I can think of.  Trying a test build now.